src/DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -19,9 +19,11 @@ public function buildConfiguration(NodeBuilder $node)
         // todo - add the comments as help text, and render in README
         $node
             ->scalarNode('access_type')
+                ->defaultValue('')
                 ->info('Optional value for sending access_type parameter. More detail: https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters')
             ->end()
             ->scalarNode('hosted_domain')
+                ->defaultValue('')
                 ->info('Optional value for sending hd parameter. More detail: https://developers.google.com/identity/protocols/OpenIDConnect#hd-param')
             ->end()
             ->arrayNode('user_fields')
```

Comment:
We need a default value (an empty array) for the `user_fields` option too, right?